module/os/freebsd/zfs/kmod_core.c

```diff
@@ -106,7 +106,7 @@ SYSCTL_INT(_vfs_zfs_version, OID_AUTO, ioctl, CTLFLAG_RD, &zfs_version_ioctl,
 
 static struct cdev *zfsdev;
 
-extern void zfs_init(void);
+extern int zfs_init(void);
```

Comment:
These three declarations are unnecessary, we might as well remove them.

Reply:
Ok, I think I did what you wanted.